mne/viz/evoked.py

```diff
@@ -276,12 +276,12 @@ def _plot_evoked(evoked, picks, exclude, unit, show, ylim, proj, xlim, hline,
                       plot_type=plot_type)
         _draw_proj_checkbox(None, params)
 
-    block = True if selectable else False
-    plt_show(show, block=block)
+    for ax in fig.axes[:-1]:
+        ax.set_xlabel(' ')
```

Comment:
I think `''` is actually better because then matplotlib can optimize the spacing better (unless they already ignore whitespace, which I doubt but is possible)